cupy/polynomial/polynomial.py

```diff
@@ -0,0 +1,24 @@
+import cupy
+
+
+def polycompanion(c):
+    """Computes the companion matrix of c.
+
+    Args:
+        c (cupy.ndarray): 1-D array of polynomial coefficients
+            ordered from low to high degree.
+
+    Returns:
+        cupy.ndarray: Companion matrix of dimensions (deg, deg).
+
+    .. seealso:: :func:`numpy.polynomial.polynomial.polycompanion`
+
+    """
+    [c] = cupy.polynomial.polyutils.as_series([c])
```

Comment:
`as_series` supports only for single `cupy.ndarray` input, so a list of `ndarray` is not guaranteed to work.

Reply:
I have a test for it and it seems to be working fine